src/main/java/sirius/kernel/xml/Outcall.java

```diff
@@ -170,6 +198,23 @@ public void setAuthParams(String user, String password) throws IOException {
         }
     }
 
+    /**
+     * Makes the underlying connection trust self-signed certs.
+     *
+     * This will make the connection trust <strong>only</strong> self-signed certificates!
```

Comment:
there is no option to accept both, self-signed and proper SSL certs?

Reply:
Currently, no. I could trust all certificates instead. There is no other easy possibility.
However, you can always add a certificate manually to the JVM Truststore, then you don't need this option at all.